packages/documentation/src/pages/frontend-support-dashboard/app.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import Layout from '../../layouts/Layout';
+
+import CommitsTable from './containers/CommitsTable';
+import { vetsWebsiteInfo, contentBuildInfo } from '../../definitions/constants';
+
+import 'web-components/dist/component-library/component-library.css';
+
+import { defineCustomElements } from 'web-components/loader';
+defineCustomElements();
+
+
+async function fetchDashboardData(repo) {
+  // https://dmitripavlutin.com/javascript-fetch-async-await/#5-parallel-fetch-requests
+  const [
+    devBuildTextResponse,
+    stagingBuildTextResponse,
+    prodBuildTextResponse,
+    commitsResponse,
+  ] = await Promise.all([
+    fetch(repo.devBuildText),
+    fetch(repo.stagingBuildText),
+    fetch(repo.prodBuildText),
+    // last 30 commits from vets-website
+    fetch(`https://api.github.com/repos/${repo.owner}/${repo.repo}/commits`),
+  ]);
+
+  const devBuildText = await devBuildTextResponse.text();
+  const stagingBuildText = await stagingBuildTextResponse.text();
+  const prodBuildText = await prodBuildTextResponse.text();
+  const commits = await commitsResponse.json();
```

Comment:
These should probably check for `response.ok` or `response.status` first, or errors might get missed.